### PR TITLE
#BL-1234 Update Banner text for July

### PR DESCRIPTION
**Pull request by cdoyle-temple**

### CODE REVIEW

File: app/views/shared/_banner.html.erb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <div class="site-message">
   <div class="container container-medium">
-    <p>
-      <%= t("blacklight.banner_part_one_html", href: link_to(t("blacklight.banner_part_one_href"), t("blacklight.banner_part_one_link"))) %>
-      <%= t("blacklight.banner_part_two_html", href: link_to(t("blacklight.banner_part_two_href"), t("blacklight.banner_part_two_link"))) %>
-    </p>
+   <div class="row">
+      <div class="col">
+        <%= image_tag "information.png", class: "decorative", alt: "" %>
```
Comment (dkinzer) [inline app/views/shared/_banner.html.erb:5]: @cdoyle-temple please add alt description to this for people who use our sight who ave visually impaired.  Thanks!

Comment (cdoyle-temple) [reply app/views/shared/_banner.html.erb:5]: Null alt descriptions should be used when the image adds no additional content to the page. This allows the visually impaired user to skip the image as decorative.

Comment (dkinzer) [reply app/views/shared/_banner.html.erb:5]: OK, I guess i'm confused.  Why is it called 'information.png' if it's just decorative in nature.  Also, I don't have an 'information.png' file on my local should it be attached to this PR?

Comment (dkinzer) [reply app/views/shared/_banner.html.erb:5]: OK, sorry didn't realize that.

**Pull request merged**